### PR TITLE
[graphql] workspaceLocationEntryOrError root resolver

### PR DESCRIPTION
**Pull request by alangenfeld**
adds a way to load workspace location entries individually by name 

## How I Tested These Changes

added test

### CODE REVIEW

**Comment by alangenfeld** [comment]
* **#21523** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/21523?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @alangenfeld and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: python_modules/dagster-graphql/dagster_graphql/implementation/external.py [hunk 1]
```diff
@@ -188,3 +189,15 @@ def fetch_location_statuses(
...
+
+
+def fetch_location_entry(
+    request_context: WorkspaceRequestContext, name: str
+) -> Optional["GrapheneWorkspaceLocationEntry"]:
+    from ..schema.external import GrapheneWorkspaceLocationEntry
+
+    location_entry = request_context.get_location_entry(name)
+    if location_entry is None:
+        return None
```
Comment (alangenfeld) [inline python_modules/dagster-graphql/dagster_graphql/implementation/external.py:201]: i opted on null on miss instead of error on miss - happy to change

**Review by salazarm — APPROVED** [review]
Nice, is the idea to fetch these in parallel for now?

**Comment by alangenfeld** [comment]
yea, this at least lets us try it out

**Pull request merged**